airflow/config_templates/config.yml

```diff
@@ -851,6 +851,13 @@ metrics:
       type: string
       example: ~
       default: ""
+    statsd_datadog_metrics_tags:
+      description: |
+        If set to True, Airflow will add metadata as tags for some of the emitted metrics
+      version_added: 2.6.0
+      type: boolean
+      example: ~
+      default: "False"
```

Comment:
Would emitting metadata introduce incompatibility? If not, I’d suggest defaulting to true instead so this feature is easier to discover.